lib/impure/db_sqlite.nim

```diff
@@ -290,7 +293,8 @@ iterator fastRows*(db: DbConn, query: SqlQuery,
     if finalize(stmt) != SQLITE_OK: dbError(db)
 
 iterator fastRows*(db: DbConn, stmtName: SqlPrepared,
-                   args: varargs[string, `$`]): Row {.tags: [ReadDbEffect,WriteDbEffect].} =
+                   args: varargs[string, `$`]): Row 
+                  {.tags: [ReadDbEffect,WriteDbEffect], since: (1, 3).} =
```

Comment:
Why are you adding `since` to this proc? It's not a new proc.

We use `since` to signify when an API is added in a new version of Nim.

Reply:
`SqlPrepared` is last PR I added